### PR TITLE
Move to kubernetes

### PR DESCRIPTION
**Pull request by lukanus**

### CODE REVIEW

**Review by christophermanning — APPROVED** [review]

File: utils/reporting/error.go [hunk 1]
```diff
@@ -17,11 +17,7 @@ func RecoverError() {
 	}
```
Comment (christophermanning) [inline utils/reporting/error.go:17]: I think the `rollbar.LogPanic(err, true)` at line 16 should be removed as well. And the `serverRoot()` func can be removed as well

File: helm/pipeline-polkadot/values.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
+global:
+  annotations:
+    figment.io/github_repository: "github.com/figment-networks/pipeline-polkadot"
```
Comment (christophermanning) [inline helm/pipeline-polkadot/values.yaml:3]:
```suggestion
    figment.io/github_repository: "github.com/figment-networks/polkadothub-indexer"
```

Comment (lukanus) [reply helm/pipeline-polkadot/values.yaml:3]: 👍

File: utils/reporting/error.go (hunk 1 above)
Comment (lukanus) [reply utils/reporting/error.go:17]: :+1

**Pull request merged**